api/search.go

```diff
@@ -132,3 +140,37 @@ func SearchHandlerFunc(queryBuilder QueryBuilder, elasticSearchClient ElasticSea
 
 	}
 }
+
+func CreateSearchIndexHandlerFunc(elasticSearchClient ElasticSearcher) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		ctx := req.Context()
+		indexName := createIndexName("ons")
+		fmt.Printf("Index created: %s\n", indexName)
+
+		indexCreated, err := elasticSearchClient.CreateNewEmptyIndex(ctx, indexName)
+		if !indexCreated {
+			if err != nil {
+				http.Error(w, serverErrorMessage, http.StatusInternalServerError)
+			}
+			return
+		}
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusCreated)
+		createIndexResponse := CreateIndexResponse{
+			IndexName: indexName}
+		jsonResponse, _ := json.MarshalIndent(createIndexResponse, "", " ")
```

Comment:
This there a technical reason we need to return formatted/pretty json for this response?

Reply:
No there isn't - my mistake - I copied the code from the web without properly understanding it. Thanks.